src/main/java/uk/gov/register/RegisterApplication.java

```diff
@@ -20,8 +19,19 @@
 import uk.gov.ida.dropwizard.logstash.LogstashBundle;
 import uk.gov.register.auth.BasicAuthFilter;
 import uk.gov.register.auth.RegisterAuthDynamicFeature;
-import uk.gov.register.configuration.*;
-import uk.gov.register.core.*;
+import uk.gov.register.configuration.ConfigManager;
```

Comment:
these verbose imports are just due to following GDS way policy when updating files: https://gds-way.cloudapps.digital/manuals/programming-languages/java.html#imports